packages/fluentui/react-northstar/src/components/Tree/Tree.tsx

```diff
@@ -353,7 +345,8 @@ export const Tree: ComponentWithAs<'div', TreeProps> &
       const { id } = treeItemProps;
       const siblings = getSiblings(stableProps.current.items, id);
 
-      setActiveItemIds(e, currActiveItemIds => {
+      setActiveItemIds(prevActiveItemIds => {
+        const currActiveItemIds = stableProps.current.activeItemIds || prevActiveItemIds;
```

Comment:
@layershifter This one is weird, I know, but I can't figure out why in the first interaction the `prevActiveItemIds`  is not equals to the `activeItemIds` passed in the props. Basically, if I have:

```jsx
<Tree activeItemIds={["1"]} />
```
When I have the first interaction with the tree ( trying to expand another item for example ) it will pass here and instead of the `prevActiveItemIds` be `["1"]`, as it was initialised,  it will be `[]`